src/soundbase.cpp

```diff
@@ -114,8 +114,9 @@ QString CSoundBase::SetDev ( const QString strDevName )
 
                 // store error return message
                 strReturn = QString ( tr ( "Can't use the selected audio device "
-                                           "because of the following error: " ) ) +
-                            strErrorMessage + QString ( tr ( " The previous driver will be selected." ) );
+                                           "because of the following error: %1 "
+                                           "The previous driver will be selected." )
```

Comment:
Not 100$ sure, but I think the error message might be long and even contain HTML. That’s why the message is at the back.

Reply:
It's no different from the older form in action so I guess it's okay?

Reply:
Argh, yes. So it’s ok probably.